app/index.html

```diff
@@ -4,19 +4,44 @@
     <meta charset="UTF-8">
     <title>Webshop</title>
     <link rel="stylesheet" href="style/index.css" type="text/css"/>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" type="text/css"/>
```

Comment:
You installed bootsrap with bower, but you didn't save the dependency to bower json (probably you didn't type --save). Please do so.